pallets/settlement/src/rng.rs

```diff
@@ -0,0 +1,118 @@
+use rand_core::{CryptoRng, Error, RngCore};
```

Comment:
If this is going to be used in multiple pallets, I guess it belongs to the primitives.

Reply:
Yeah for sure. Will do.

Reply:
https://polymath.atlassian.net/browse/CRYP-174